server/log/main.go

```diff
@@ -133,6 +157,16 @@ func main() {
 		glog.Fatalf("Failed to load server key: %v", err)
 	}
 
+	// Start HTTP server (optional)
+	if *exportRpcMetrics {
```

Comment:
I think the gRPC server contains a HTTPServer already, do you need another one (I'm not sure what witchcraft is done behind the scenes to make the expvars appear there though.)